packages/terra-site/src/examples/progress-bar/ProgressBarGradient.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ProgressBar from 'terra-progress-bar';
+
+const ProgressBarDefault = () => (
```

Comment:
This is pretty minor, but I think this variable was copy pasted over. `ProgressBarGradient` may be more descriptive of this component.

Reply:
@StephenEsser Updated. Thanks.